### ISSUE TITLE
Some error about validation loss.

### ISSUE THREAD

**Issue by supyer9** [ifzhang/FairMOT#167]
![image](https://user-images.githubusercontent.com/54097420/85981141-48b3a100-ba16-11ea-9a29-e7d0413ad43d.png)
![image](https://user-images.githubusercontent.com/54097420/85981178-536e3600-ba16-11ea-8810-cd4c0eeb0b62.png)
![image](https://user-images.githubusercontent.com/54097420/85981196-59fcad80-ba16-11ea-8952-7e25c3bef9d8.png)
![image](https://user-images.githubusercontent.com/54097420/85981208-61bc5200-ba16-11ea-9bf0-b4a00a02a7f4.png)
Hello！I modified your code so that it can verify the validation set after training an epoch training set. Although the loss curve of the training set looks normal, there are some problems with the loss curve of the verification set. The heatmap verification loss curve does not decrease but rises, and offset loss and wh loss are in a downward trend.
I ensure that the code after my modification is no problem, and the data distribution of the training set and the validation set are similar, can you help me see where the problem is? thank you very much

**Comment by ifzhang** [comment]
It's a little bit strange. May I ask what training set and validation set you use?

**Comment by supyer9** [comment]
@ifzhang Thank you for your reply. 
The data set I use is UA-detrac. Is it possible that the model was already overfitted during the first epoch, so the validation set rose directly from the beginning?
But the trend of heat_map_loss and wh_loss and offset_loss of the verification set are different, is this normal?

**Comment by zjutwt** [comment]
hi could you share your code which verify the validation set,thanks a lot @supyer9
